### PR TITLE
add information about contributing

### LINKED ISSUE #4: document guidelines for contributing
i suggest we enforce a fork and pull model for this repository (or at a bare minimum require submitting Pull Requests to merge into master from dedicated feature branches) and explicitly prohibit that contributors push directly to the master branch.

https://help.github.com/articles/using-pull-requests/

this will be beneficial for several reasons

1. a second set of eyes helps spot errors and gives an opportunity to improve both code and documentation quality.
2. it helps avoid merge conflicts
3. its good practice for contributing to strangers projects and working on a codebase which requires more extensive collaboration.

its also important that we document guidelines that explain what is required in order to get contributions merged.  particularly the inclusion of enough contextual information in a readme specific to an individual sample to explain to anyone perusing the repository what it should be able to help them accomplish.

### PR DESCRIPTION
**Pull request by jgravois**
by no means 'done', but resolves #4

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+### Before filing an issue
+
+If you're just looking for help, you'll probably attract the most eyes if you post in [GIS Stackexchange](http://gis.stackexchange.com/questions/ask?tags=esri-leaflet,leaflet) or [GeoNete](https://geonet.esri.com/community/discussions-lobby).
+
+### I want to contribute, how do i go about it?
+
+in general, we enforce a [fork and pull](https://help.github.com/articles/using-pull-requests/) model for new contributions to this repository, both for members of the community at large, and for our core contributing team.
+
+if you're new to forking, you can find a pretty excellent resource to get more familiar with the general steps [here](https://guides.github.com/activities/contributing-to-open-source/).
```
Comment (nohe427) [inline CONTRIBUTING.md]: Capitalize if here.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+Esri welcomes contributions from anyone and everyone. Please see our [guidelines for contributing](https://github.com/esri/contributing).
+
+### Before filing an issue
+
+If you're just looking for help, you'll probably attract the most eyes if you post in [GIS Stackexchange](http://gis.stackexchange.com/questions/ask?tags=esri-leaflet,leaflet) or [GeoNete](https://geonet.esri.com/community/discussions-lobby).
+
+### I want to contribute, how do i go about it?
+
+in general, we enforce a [fork and pull](https://help.github.com/articles/using-pull-requests/) model for new contributions to this repository, both for members of the community at large, and for our core contributing team.
```
Comment (nohe427) [inline CONTRIBUTING.md]: Capitalize in here

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+if you're new to forking, you can find a pretty excellent resource to get more familiar with the general steps [here](https://guides.github.com/activities/contributing-to-open-source/).
+
+feel free to file an [issue](https://github.com/Esri/developer-support/issues/new) if you get stuck, we're happy to help.
```
Comment (nohe427) [inline CONTRIBUTING.md]: Capitalize feel here.

**Comment by jgravois** [comment]
aye aye cap'n.  patched and squashed.

**Comment by nohe427** [comment]
@jgravois I sent a few revisions to your email for the read me.  Let me know what you think.

**Comment by jgravois** [comment]
please keep the discussion in the PR.  

i think this is a good opportunity for some other folks who have expressed interest to get their hands dirty, so perhaps we can merge what i've got so far.  Afterward you can log another issue with explicit recommendations and we can make further refinements in another Pull Request.

are you interested in helping @summelanie @kgerrow @marlak @jguard ?  anyone else?

**Pull request merged**